webapp/src/pages/Settings.tsx

```diff
@@ -1,78 +1,699 @@
+              {resultingConfig.columns?.label}
+            </Typography>
+          </Box>
+        </Box>
+      </Box>
+      {divider}
+      {displaySectionTitle("dataset")}
+      <Box
+        display="flex"
+        flexDirection="row"
+        justifyContent="flex-start"
+        alignContent="start"
+        flexWrap="wrap"
+        gap={(theme) => theme.spacing(1, 5)}
+        marginLeft={2}
+      >
+        {displayReadonlyFields(
+          "class_name",
+          resultingConfig.dataset?.class_name
+        )}
+        {displayReadonlyFields("remote", resultingConfig.dataset?.remote)}
+        {resultingConfig.dataset?.kwargs &&
+          displayArgumentsList("kwargs", resultingConfig.dataset.kwargs)}
+        {resultingConfig.dataset?.args &&
+          resultingConfig.dataset.args.length > 0 &&
+          displayArgumentsList("args", resultingConfig.dataset.args)}
+      </Box>
+    </Box>
+  );
+  const getModelContractConfigSection = () => (
+    <Box display="flex" flexDirection="column" justifyContent="flex-start">
+      {displaySectionTitle("general")}
+      <Box
+        display="flex"
+        flexDirection="row"
+        justifyContent="flex-start"
+        marginLeft={2}
+        gap={5}
+      >
+        {displayReadonlyFields(
+          "model_contract",
+          resultingConfig.model_contract
+        )}
+        {displayReadonlyFields(
+          "saliency_layer",
+          resultingConfig.saliency_layer
+        )}
+        <Box display="flex" flexDirection="column" paddingTop={1}>
+          <Typography variant="caption">uncertainty</Typography>
+          {resultingConfig.uncertainty &&
+            _.sortBy(Object.entries(resultingConfig.uncertainty)).map(
+              ([field, value], index) => (
+                <Box key={index} display="flex" flexDirection="row">
+                  <Typography
+                    sx={{
+                      s: 1,
+                      width: "20ch",
+                      whiteSpace: "normal",
+                    }}
+                    variant="body2"
+                  >
+                    {field}:
+                  </Typography>
+                  <TextField
+                    id={field}
+                    sx={{ s: 1, width: "5ch" }}
+                    size="small"
+                    type="number"
+                    value={value}
+                    disabled={!resultingConfig.uncertainty}
+                    style={{ fontSize: 14 }}
+                    inputProps={{
+                      ...STEPPER[field],
+                      style: { fontSize: 14 },
+                    }}
+                    variant="standard"
+                    onChange={(event) =>
+                      setPartialConfig({
+                        ...partialConfig,
+                        uncertainty: _.assign({}, resultingConfig.uncertainty, {
+                          [field]: Number(event.target.value),
+                        }),
```

Comment:
As seen in my suggestion https://github.com/ServiceNow/azimuth/pull/282#discussion_r1042739485, you can get rid of lodash completely, here:
```suggestion
                        uncertainty: {
                          ...resultingConfig.uncertainty,
                          [field]: Number(event.target.value),
                        },
```